lib/dmesg.rb

```diff
@@ -379,17 +379,17 @@ def generate_error_id(line)
   error_id.gsub!(/\b[0-9a-f]{8}\b/, '#')
   error_id.gsub!(/\b[0-9a-f]{16}\b/, '#')
   error_id.gsub!(/(=)[0-9a-f]+\b/, '\1#')
-  error_id.gsub!(/[+\/]0x[0-9a-f]+\b/, '')
-  error_id.gsub!(/[+\/][0-9a-f]+\b/, '')
+  error_id.remove!(/[+\/]0x[0-9a-f]+\b/)
+  error_id.remove!(/[+\/][0-9a-f]+\b/)
```

Comment:
thanks, help merge multiple remove!, like remove!(pattern1, pattern2). Similar comments to others.